prepare.gasket.patch

```diff
@@ -1,17 +1,13 @@
 diff --git kernel/prepare/pkg.yaml kernel/prepare/pkg.yaml
-index 37b01e3..5615ff9 100644
+index 37b01e3..441ce61 100644
 --- kernel/prepare/pkg.yaml
 +++ kernel/prepare/pkg.yaml
-@@ -40,6 +40,12 @@ steps:
+@@ -40,6 +40,8 @@ steps:
          done
  
          make mrproper
 +      - |
-+        set -e
-+        patch -p0 < /pkg/patches/gasket.patch
-+        grep 'gasket' drivers/Kconfig
-+        grep 'gasket' drivers/Makefile
-+        ls -la drivers/staging/gasket
++        /pkg/scripts/add-gasket.sh
```

Comment:
lol...this was very confusing to read...a diff of a diff 🤯 . I kept thinking it had me as adding all those lines, but in reality i removed them all and replaced it with `/pkg/scripts/add-gasket.sh`.